webapp/src/components/user_attribute/user_attribute.jsx

```diff
@@ -8,6 +8,7 @@ const {formatText, messageHtmlToComponent} = window.PostUtils;
 export default class UserAttribute extends React.PureComponent {
     static propTypes = {
         id: PropTypes.string.isRequired,
+        postIsWebhook: PropTypes.string.isRequired,
```

Comment:
Nit pick: Maybe use the same naming as the Prop 
```suggestion
        fromWebhook: PropTypes.string.isRequired,
```

Reply:
Addressed in the next commit :)